cv32e40s/tests/programs/custom/illegal_access_loop_tests/csr_privilege_gen.py

```diff
@@ -0,0 +1,80 @@
+"""
+script to generate the csr_register_loop
+
+"""
+
+input_string = "// Start of generated code." # start string, this will move the wrie HEAD
+pointer = 0 # file pointer 
+filename = "csr_privilege_loop.S" # file which will be written to 
+
+
+# register list fetched manually from the spec (V20211203), contains all U-, S-, R-, and M-mode CSR registers
+reg_str = """
+0x100-0x1FF
```

Comment:
This starts at `0x100`, are the user level CSRs missing?

Reply:
From discussions in chat: 0x000 etc are deliberately left out because this test intends to do U-mode accesses to "MRW" CSRs etc.

Could we add this in a comment too?

I read "contains all U-, S-, R-, and M-mode CSR registers" and wondered why U was missing.

Reply:
Yeah, I can change this locally now before pushing the new change. It should be just S-, R-, and M-mode CSR's

Reply:
This is fixed now

Reply:
Thanks. If it is fixed then it should be ok to push the "Resolve conversation" button if it exists.